packages/manager/apps/web/client/app/hosting/dashboard/hosting.html

```diff
@@ -145,46 +145,119 @@ <h1 class="d-inline-block mb-0 word-break">
         </div>
     </div>
     <div class="oui-header__tabs">
-        <!-- Adapted header-tabs for the old use. Need to refactor hosting routing afterwards -->
         <oui-header-tabs>
             <oui-header-tabs-item
-                data-ng-if="$ctrl.tabs"
-                data-ng-repeat="tab in $ctrl.tabs"
-                data-href="{{:: $ctrl.$state.href('app.hosting', { 'tab': tab }) }}"
-                data-active="$ctrl.selectedTab === tab"
-                data-on-click="$ctrl.setSelectedTab(tab)"
-                ><span
-                    data-ng-bind=":: 'hosting_tab_' + tab | translate"
-                ></span>
+                href="{{ $ctrl.generalInformationLink }}"
+                active="$ctrl.currentActiveLink() === $ctrl.generalInformationLink"
+            >
+                <span data-translate="hosting_tab_GENERAL_INFORMATIONS"></span>
+            </oui-header-tabs-item>
+            <oui-header-tabs-item
+                href="{{ $ctrl.multisiteLink }}"
```

Comment:
one time binding here an also in all other tabs?

Reply:
See my previous comment :)